apps/web/src/pages/templates/workflow/workflow/node-types/WorkflowNode.tsx

```diff
@@ -336,6 +336,12 @@ const StyledContentWrapper = styled.div`
   justify-content: flex-start;
   width: 100%;
   flex: 1;
+    .workflow-node-subtitle {
+    overflow: 'hidden',
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+    maxWidth: '60%',
+  }
```

Comment:
workflow-node-subtitle is a data-test-id. Styles could not be applied on that id, these styles will have no affect on the node content.